changelogs/fragments/533-postgresql_tablespace.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
+- "postgresql_tablespace - improved autocommit handling (https://github.com/ansible-collections/community.postgresql/pull/533)."
```

```suggestion
minor_changes:
- "postgresql_tablespace - improved autocommit handling (https://github.com/ansible-collections/community.postgresql/pull/533)."
```
as this is just refactoring, could you please remove the fragment?